debian/check_control

```diff
@@ -17,20 +19,24 @@ set -e
 
 cd "$(dirname "$0")/.."
 
-# LIBNO-file contains "LIBNO = 24.1.1"
-# VERSION contains dots 24.1.2
-# LIBNO contains dashes 24-1-2
-VERSION=$(grep LIBNO LIBNO | sed -e "s/LIBNO = //g")
-LIBNO=$(echo "${VERSION}" | sed -e "s/\./-/g")
+# LIBNO-file can contain "LIBNO = 24.1.1-oracular1" (The part after x.y.z is optional)
+# VERSION contains dots: 24.1.2-oracular1
+VERSION=$(grep LIBNO LIBNO | sed -e "s/LIBNO *= *//g")
```

Comment:
I think we can safely assume that we have only one line in LIBNO, otherwise meson will probably also explode. Also strip whitespaces:
```suggestion
VERSION=$(sed -e "s/LIBNO\s*=\s*//g")
```

Reply:
Fixed suggestion  and applied.